.github/workflows/push-build.yaml

```
@@ -0,0 +1,31 @@
+name: Build Image
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+    tags:
+      - v*
+jobs:
+  push-master-image-on-docker:
```

```suggestion
  push-image-to-container-registry:
```

Let's make it generic.

---

done.